### PR TITLE
Partial (2/2) spanish translation ch08-03-hash-maps.md

### PR DESCRIPTION
**Pull request by jossue33**
Number of words: 810
Line: 184-299

### CODE REVIEW

File: second-edition/src/ch08-03-hash-maps.md [hunk 1]
```diff
@@ -249,51 +249,51 @@ for word in text.split_whitespace() {
...
+* Dada una lista de enteros, usa un vector y regresa la media (promedio), mediana
+ (cuando está ordenado, el valor en la posición media) y modo (el valor que
+  ocurre más a menudo; un mapa hash será útil aquí) de la lista.
+* Convierte cadenas para cerdo latino. La primera consonante de cada palabra se mueve a
+  se agrega el final de la palabra y "ay", por lo que "primero" se convierte en "irst-fay". Palabras
+  que comienzan con una vocal tienen "hay" agregado al final en su lugar ("apple" se convierte
+  “apple-hay”). ¡Tenga en cuenta los detalles sobre la codificación UTF-8!
+* Usando un mapa hash y vectores, crea una interfaz de texto para permitir que un usuario agregue
+  nombres de empleados a un departamento en una empresa. Por ejemplo, "Agregar a Sally a
+  Ingeniería "o" Agregar Amir a Ventas ". Luego, deje que el usuario recupere una lista de todos
```
Comment (thecodix) [inline second-edition/src/ch08-03-hash-maps.md]: 'todas las personas'

File: second-edition/src/ch08-03-hash-maps.md [hunk 2]
```diff
@@ -249,51 +249,51 @@ for word in text.split_whitespace() {
...
+otros usuarios de Rust que proporcionan hashers implementando muchos algoritmos 
+hash comúnes.
+
+## Resúmen
+
+Los vectores, cadenas y mapas hash proporcionarán una gran cantidad de funcionalidades
+que necesita en programas donde necesita almacenar, acceder y modificar datos.
+Aquí hay algunos ejercicios que ahora debes estar preparado para resolver:
+
+* Dada una lista de enteros, usa un vector y regresa la media (promedio), mediana
```
Comment (thecodix) [inline second-edition/src/ch08-03-hash-maps.md]: 'regresa' might be better as 'devuelve'

File: second-edition/src/ch08-03-hash-maps.md [hunk 3]
```diff
@@ -181,22 +181,22 @@ scores.insert(String::from("Blue"), 25);
 println!("{:?}", scores);
 ```
 
-<span class="caption">Listing 8-24: Replacing a value stored with a particular
-key</span>
+<span class="caption">Listado 8-24: Reemplazar un valor almacenado con una llave
```
Comment (thecodix) [inline second-edition/src/ch08-03-hash-maps.md]: 'clave'

**Comment by jossue33** [comment]
@thecodix Todo esta listo

**Comment by thecodix** [comment]
Thanks!

**Comment by thecodix** [comment]
machine translated

**Pull request merged**